Xamarin.Forms.Platform.Android/CollectionView/RecyclerViewScrollListener.cs

```diff
@@ -76,31 +74,28 @@ public override void OnScrolled(RecyclerView recyclerView, int dx, int dy)
 			}
 		}
 
-		static int CalculateCenterItemIndex(int firstVisibleItemIndex, int lastVisibleItemIndex, LinearLayoutManager linearLayoutManager)
+		static int CalculateCenterItemIndex(int firstVisibleItemIndex, RecyclerView recyclerView, LinearLayoutManager linearLayoutManager)
```

Comment:
we need to pass the firstvfisibleItemIndex? we can grab it from linearLayoutManager i think